draft-demarco-oauth-status-attestations.md

```diff
@@ -393,9 +394,10 @@ Content-Type: application/json
 ## Credential Issuer Metadata
 
 The Credential Issuers that uses the Status Attestations MUST include in their
-OpenID4VCI [@!OpenID.VCI] metadata the claim `status_attestation_endpoint`,
-which its value MUST be an HTTPs URL indicating the endpoint where
-the Wallet Instances can request Status Attestations.
+OpenID4VCI [@!OpenID.VCI] metadata the claims:
+
+- `status_attestation_endpoint`. REQUIRED. It MUST be an HTTPs URL indicating the endpoint where the Wallet Instances can request Status Attestations.
+- `credential_hash_alg_supported`. REQUIRED. The supported Algorithm used by the Wallet Instance to hash the Digital Credential for which the Status Attestation is requested. The value SHOULD be set to `S256`.
```

Comment:
SD-JWT uses the IANA registry: https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-07#section-5.1.1

Reply:
moved in the issue https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/16